pkg/cmd/factory/factory.go

```diff
@@ -52,6 +56,17 @@ type ClientFlags struct {
 
 var _ ClientGetter = &ClientFlags{}
 
+func (c *ClientFlags) ToRESTConfig(ownerAddr string) (*rest.Config, error) {
+	clientConfig := &rest.Config{}
+	credential := c.GetCredential()
+	clientConfig.Host = ownerAddr
+	clientConfig.Credential = credential
+	// TODO: make it configurable
+	clientConfig.RateLimiter = rate.NewLimiter(rate.Every(100*time.Millisecond), 10)
```

Comment:
Can you explain the purpose of the RateLimiter here in comments?
Can you eliminate the magic number `100`?